includes/class-blocks.php

```diff
@@ -186,8 +186,21 @@ public static function render_post_reactions_block( $attrs, $content ) {
 		}
 
 		$args = array( 'data-attrs' => wp_json_encode( $attrs ) );
-		if ( isset( $attrs['title'] ) ) {
-			$args['class'] = 'activitypub-reactions-block';
+		if ( empty( $content ) ) {
+			if ( isset( $attrs['title'] ) ) {
+				$content = '<div class="activitypub-reactions"><div class="activitypub-reactions-block" ' . \get_block_wrapper_attributes( $args ) . '></div></div>';
+			} else {
+				$has_comments = \get_comments(
```

Comment:
Should this be checked even if you have a custom reactions block title?

Reply:
In the case of a custom title `view.js` handles displaying that. However there's no good way to determine whether it uses a heading block or not from in there. At least not in this version of the block.